spec/helpers/application_helper/toolbar_builder_spec.rb

```diff
@@ -1149,7 +1149,7 @@ def method_missing(sym, *args)
 
         it "record is cloneable" do
           @record = Vm.create(:type => "ManageIQ::Providers::Redhat::InfraManager::Vm", :name => "rh", :location => "l1", :vendor => "redhat")
```

Comment:
`Redhat` is not cloneable.. You want `Vmware` here, and presumably keep `be_falsey`?